scRNA-seq-advanced/setup/README.md

```diff
@@ -7,11 +7,55 @@ These files should include source locations for downloading raw data as appropri
 
 ## File locations
 
-On the RStudio server the main location for the files needed for this training module will be `/shared/data/training-modules/scRNA-seq-advanced/`.
+On the RStudio server the main location for the files needed for this training module will be `/shared/data/training-modules/scRNA-seq-advanced/data`.
 The files are then organized by dataset.
 
-After setup, the symlinks should be established from `/shared/data/training-modules/scRNA-seq-advanced/` to `training-modules/scRNA-seq-advanced/data` as appropriate.
+After setup, the symlinks should be established from `/shared/data/training-modules/scRNA-seq-advanced/data` to `training-modules/scRNA-seq-advanced/data` as appropriate.
 These links will usually be created using the script at `training-modules/scripts/link-data.sh`, so directories and files should be added to the `link_locs` array in that script.
 If no data will be written to a data directory, linking to that directory will be sufficient, but in some cases links to individual files may be required.
 
 
+## Data Sets
```

Comment:
I think we can add `RMS` integration data here at this point, since the setup notebook is already merged. Alternatively I can circle back to this once its workflow is wrapped up.

Reply:
Let's add that with the workflow.